docs/guides/burst_insar_product_guide.md

```diff
@@ -0,0 +1,290 @@
+- topsapp.swap_burst_vrts(): Switch the reference and secondary bursts to use the burst data download from ASF
+- topsapp.run_topsapp_burst(start='computeBaselines', end='unwrap2stage', config_xml=config_path):
+    Run the remaining processing steps including:
+    - Calculate the perpendicular and parallel baselines
+    - Verify the DEM file to make sure it covers the bursts
+    - Map DEM into the radar coordinates of the reference image. This generates the longitude,
+            latitude, height and LOS angles on a pixel by pixel grid for each burst.
+    - Estimate the azimuth offsets between the input SLC bursts (The Enhanced Spectral Diversity (ESD) method is NOT used)
+    - Estimate the range offsets between the input SLC bursts
+    - Coregister the secondary SLC burst by applying the estimated range and azimuth offsets
+    - Produce the wrapped phase interferogram
+    - Unwrap the wrapped phase interferogram using SNAPHU to produce the wrapped phase interferogram
+- topsapp.run_topsapp_burst(start='geocode', end='geocode', config_xml=config_path): Geocode the output products
+
+### Post-Processing
+
+#### Apply Water Mask
+A water mask identifying coastal waters and major inland waterbodies is generated using the Global Self-consistent,
+Hierarchical, High-resolution Geography Database (GSHHG) dataset (https://www.ngdc.noaa.gov/mgg/shorelines). This water mask raster is always included with the Burst InSAR products for reference, but is not applied to the interferometry products by default.
+
+Users can optionally choose to apply the water mask to output products, which affects the wrapped interferogram,
+the unwrapped interferogram, and the browse image. Areas covered by the water mask in these output images are set to
+NoData.
+
+Applying a water mask to an interferogram is only supported *after* phase unwrapping. Note that
+applying the mask after phase unwrapping does not prevent unwrapping errors caused by the inclusion of water pixels
+as valid data during the phase unwrapping process. When phase unwrapping occurs over large expanses of water, it can
+lead to unexpected deformation signals or phase jumps in the unwrapped outputs, and the current masking approach
+does not correct for these impacts.
+
+#### Product Creation
+Image files are exported into the widely-used GeoTIFF format in a Universal Transverse Mercator (UTM) projection. Images
+are resampled to a pixel size that reflects the resolution of output image based on the requested number of looks: 80m
+pixel size for 20x4 looks, 40m for 10x2 looks, and 20m for 5x1 looks.
+
+Supporting metadata files are created, as well as a quick-look browse image.
+
+{% endblock %}
+
+{% block packaging %}
+## Product Packaging
+
+HyP3 Burst InSAR output is a zip file containing various files, including GeoTIFFs, PNG browse images with geolocation information, Google Earth KMZ files, a metadata file, and a README file.
```

Comment:
```suggestion
HyP3 Burst InSAR output is a zip file containing various files, including GeoTIFFs, a PNG browse image, a metadata file, and a README file.
```
We don't include a KMZ or multiple browse images for burst products like we do for full insar products.